documentation/modules/deploying/proc-using-access-operator.adoc

```diff
@@ -0,0 +1,130 @@
+
+* If only one listener is configured in the `Kafka` resource, that listener is selected.
+* If multiple listeners are configured, the operator selects one with the appropriate security based on tls and authentication properties in the `Kafka` and `KafkaUser` resources.
+* If multiple listeners with appropriate security are available, the operator selects an internal listener, if present.
+* If multiple internal listeners with appropriate security are available, the operator selects the first listener alphabetically by name.
+
+.Prerequisites
+
+* A Kafka cluster deployed  using a `Kafka` resource and managed by the Cluster Operator.
+* `KafkaUser` resources must be created for any users needing authentication to the Kafka cluster.
+
+Example `KafkaAccess` resource specifications are contained in the `./examples` directory of Strimzi's link:https://github.com/strimzi/kafka-access-operator[`kafka-access-operator` GitHub repository^].
+
+.Procedure
+
+. Configure a `KafkaAccess` custom resource to bind your applications to the Kafka cluster. 
++
+** Specify the name of the `Kafka` resource to connect to the Kafka cluster.
+** (Optional) Specify the name of the listener used to connect to the Kafka cluster.
+** Specify the `KafkaUser` resources representing users requiring access to the Kafka cluster.
++
+Make sure the name, namespace and listener in the `KafkaAccess` custom resource match those of your Kafka instance.
++
+.Example access configuration
+[source,yaml]
+----
+apiVersion: access.strimzi.io/v1alpha1
+kind: KafkaAccess
+metadata:
+  name: my-kafka-access
+spec:
+  kafka:
+    name: my-cluster
+    namespace: kafka
+    listener: tls
+  user:
+    kind: KafkaUser
+    apiGroup: kafka.strimzi.io
+    name: my-user
+    namespace: kafka
+----
+
+. Create the `KafkaAccess` resource in Kubernetes:
++
+[source,shell]
+----
+kubectl apply -f <kafka_access_config_file> -n strimzi-access-operator
+----
+
+. Check the status of the `KafkaAccess` custom resource: 
++
+[source,shell]
+----
+kubectl get kafkaaccess my-kafka-access -o yaml
+----
++
+The status of the resource is updated once the operator creates the binding `Secret`. 
++
+.Status shows binding secret created 
+[source,yaml]
+----
+# ...
+status: 
+  binding: 
+    name: 
+      kafka-binding
+----
+
+. Inspect the created binding secret: 
++ 
+[source,shell]
+kubectl get secret kafka-binding -o yaml
++
+The secret contains connection details and user credentials in various formats to accommodate the requirements of different development environments and applications.
++
+.Example binding secret
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kafka-binding
+type: servicebinding.io/kafka
+data:
+  type: kafka
+  provider: strimzi
+  # Kafka cluster connection properties # <1>
+  bootstrap.servers: my-cluster-kafka-bootstrap:9092
+  bootstrap-servers: my-cluster-kafka-bootstrap:9092  
+  bootstrapServers: my-cluster-kafka-bootstrap:9092
```

Comment:
my ignorance here about access operator, is it normal having the 3 different fields for the bootstrap servers (with different naming) ... I guess same applies to other fields within this Secret. @katheris ?

Reply:
@ppatierno I know this was already merged, but just to answer your qn we chose to give a few options so that applications using frameworks like Quarkus or Spring would work out of the box. So that's the reason for the different names

Reply:
@katheris thanks! What do you think about having comments in the above doc to explain why. Just to avoid silly questions coming from other people like me :-)